stdlib/public/SDK/Foundation/AffineTransform.swift

```diff
@@ -161,17 +166,24 @@ public struct AffineTransform : ReferenceConvertible, Hashable, CustomStringConv
      Mutates an affine transformation matrix from a rotation value (angle α in radians).
      The matrix takes the following form:
      
-         [  cos α   sin α  0 ]
-         [ -sin α   cos α  0 ]
-         [    0       0    1 ]
+     [  cos α   sin α  0 ]
+     [ -sin α   cos α  0 ]
```

Comment:
Yep. Copy/paste into an autoindenting environment shouldn't be a be-all, end-all goal because autoindent isn't perfect.